extensions/notebook/src/jupyter/jupyterNotebookProvider.ts

```diff
@@ -64,13 +64,15 @@ export class JupyterNotebookProvider implements nb.NotebookProvider {
 				manager.sessionManager.shutdown(session.id);
 			}
 			if (sessionManager.listRunning().length === 0) {
-				const TenMinutesInMs = 10 * 60 * 1000;
+				let notebookConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(constants.notebookConfigKey);
```

Comment:
I'm thinking we should have 0 be "never shuts down" as well (no timeout)

Reply:
I think that makes sense. The servers will get shutdown once ADS is closed so there shouldn't be any issues with not having a timeout.